konduit-serving-data/konduit-serving-image/src/test/java/ai/konduit/serving/data/JsonCoverageTest.java

```diff
@@ -0,0 +1,275 @@
+ *  * https://www.apache.org/licenses/LICENSE-2.0.
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  * License for the specific language governing permissions and limitations
+ *  * under the License.
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *  *****************************************************************************
+ */
+package ai.konduit.serving.data;
+
+import ai.konduit.serving.common.test.BaseJsonCoverageTest;
+import ai.konduit.serving.data.image.convert.ImageToNDArrayConfig;
+import ai.konduit.serving.data.image.convert.config.AspectRatioHandling;
+import ai.konduit.serving.data.image.convert.config.ImageNormalization;
+import ai.konduit.serving.data.image.convert.config.NDChannelLayout;
+import ai.konduit.serving.data.image.convert.config.NDFormat;
+import ai.konduit.serving.data.image.step.bb.extract.ExtractBoundingBoxStep;
+import ai.konduit.serving.data.image.step.capture.CameraFrameCaptureStep;
+import ai.konduit.serving.data.image.step.capture.VideoFrameCaptureStep;
+import ai.konduit.serving.data.image.step.crop.ImageCropStep;
+import ai.konduit.serving.data.image.step.face.DrawFaceKeyPointsStep;
+import ai.konduit.serving.data.image.step.grid.crop.CropFixedGridStep;
+import ai.konduit.serving.data.image.step.grid.crop.CropGridStep;
+import ai.konduit.serving.data.image.step.grid.draw.DrawFixedGridStep;
+import ai.konduit.serving.data.image.step.grid.draw.DrawGridStep;
+import ai.konduit.serving.data.image.step.ndarray.ImageToNDArrayStep;
+import ai.konduit.serving.data.image.step.point.convert.RelativeToAbsoluteStep;
+import ai.konduit.serving.data.image.step.point.draw.DrawPointsStep;
+import ai.konduit.serving.data.image.step.point.heatmap.DrawHeatmapStep;
+import ai.konduit.serving.data.image.step.point.perspective.convert.PerspectiveTransformStep;
+import ai.konduit.serving.data.image.step.resize.ImageResizeStep;
+import ai.konduit.serving.data.image.step.segmentation.index.DrawSegmentationStep;
+import ai.konduit.serving.data.image.step.show.ShowImageStep;
+import ai.konduit.serving.pipeline.api.data.NDArrayType;
+import ai.konduit.serving.pipeline.api.data.Point;
+import ai.konduit.serving.pipeline.util.ObjectMappers;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class JsonCoverageTest extends BaseJsonCoverageTest {
+
+    private ImageToNDArrayConfig c = new ImageToNDArrayConfig()
+            .height(256)
+            .width(256)
+            .channelLayout(NDChannelLayout.RGB)
+            .includeMinibatchDim(true)
+            .format(NDFormat.CHANNELS_LAST)
+            .dataType(NDArrayType.UINT8)
+            .normalization(new ImageNormalization(ImageNormalization.Type.SUBTRACT_MEAN));
```

Comment:
For ImageNormalization - we should also test when meanRgb, stdRgb and maxValue are set